test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1

```diff
@@ -46,123 +48,181 @@ Describe "DSC MOF Compilation" -tags "CI" {
+        }
+
+
+        #Ensure directory for Wordpress site
+        nxFile wpHttpDir{
+            DestinationPath = "/var/www/wordpress"
+            Type = "Directory"
+            Ensure = "Present"
+            Mode = "755"
+        }
+
+        #Ensure share directory
+        nxFile share{
+            DestinationPath = "/mnt/share"
+            Type = "Directory"
+            Ensure = "Present"
+            Mode = "755"
+        }
+
+        #Bind httpd to port 8080
+        nxFile HttpdPort{
+           DestinationPath = "/etc/httpd/conf.d/listen.conf"
+           Ensure = "Present"
+           Contents = "Listen 8080`n"
+           Type = "File"
+        }
+
+        #nfs mounts
+        nxScript nfsMount{
+            TestScript= "#!/bin/bash"
+            GetScript="#!/bin/bash"
+            SetScript="#!/bin/bash"
+
+        }
+
+        #Retrieve latest wordpress
+        nxFile WordPressTar{
+            SourcePath = "/mnt/share/latest.zip"
+            DestinationPath = "/tmp/wordpress.zip"
+            Checksum = "md5"
+            Type = "file"
+            DependsOn = "[nxScript]nfsMount"
+        }
+
+        #Extract wordpress if changed
+        nxArchive ExtractSite{
+            SourcePath = "/tmp/wordpress.zip"
+            DestinationPath = "/var/www/wordpress"
+            Ensure = "Present"
+            DependsOn = "[nxFile]WordpressTar"
+         }
+
+         #Set wp-config
+
+
+         #Fixup SE Linux context
+         #nxScript SELinuxContext{
+            #TestScript= "#!/bin/bash"
+            #GetScript = "#!/bin/bash"
+            #SetScript = "#!/bin/bash"
+         #}
+
+         #Disable SELinux
+         nxFileLine SELinux {
+            Filepath = "/etc/selinux/config"
+            DoesNotContainPattern = "SELINUX=enforcing"
+            ContainsLine = "SELINUX=disabled"
+         }
+
+
+        nxScript SELinuxHTTPNet{
+          GetScript = "#!/bin/bash`ngetsebool httpd_can_network_connect"
+          setScript = "#!/bin/bash`nsetsebool -P httpd_can_network_connect=1"
+          TestScript = "#!/bin/bash`n exit 1"
+        }
+
+
+
+        }
+
+    }
         WordPressServer
-"@).Invoke()
+"@) | should not throw
+
+        ".\WordPressServer\CentOS.mof" | Should Exist
 
         Remove-Item -Force -Recurse -Path WordPressServer
     }
 
+    It "Should be able to compile a MOF from a basic configuration on Windows" -Skip:($IsMacOS -or $IsLinux) {
+        [Scriptblock]::Create(@"
+        configuration DSCTestConfig
+        {
+            Import-DscResource -ModuleName PSDesiredStateConfiguration
+            Node "localhost" {
+                File f1
+                {
+                    DestinationPath = "$env:SystemDrive\\Test.txt";
+                    Ensure = "Present"
+                }
+            }
+        }
+
+        DSCTestConfig
+"@) | should not throw
+
+        ".\DSCTestConfig\localhost.mof" | Should Exist
+
+        Remove-Item -Force -Recurse -Path DSCTestConfig
```

Comment:
The remove should be in an `AfterEach` or the `finally` or a `try{}finally{}`  otherwise it won't run if the test fails.  Might be easier to just use the test drive...

Reply:
I've submitted a PR to resolve this:  https://github.com/Indhukrishna/PowerShell/pull/2